examples/collect_diamonds/README.md

```diff
@@ -114,6 +116,134 @@ The three optimal action plans required to get a reward are:
 ) ; [8a298365b46b204b][1]
 ```
 
+2. Polyaction plans
+
+2.1.`(outside(self, house) ∧ do(go_to_key) ≺ do(go_to_house)) ≺ do(go_to_diamonds) ↝ Reward(1)`
```

Comment:
@tanksha, could you replace `≺` by `⩘`?  This symbol is used throughout the code.